viewer/packages/rendering/src/glsl/post-processing/pure-depth-ssao.frag

```diff
@@ -69,23 +69,23 @@ void main(){
   vec3 viewNormal = computeWorldNormalFromDepth(tDepth, vec2(float(textureSize.x), float(textureSize.y)), vUv, d);
 
   vec3 viewPosition = viewPosFromDepth(d, vUv);
+  float distanceFactor = max(length(viewPosition) / 20.0, 1.0);

```

Comment:
Can we pull out this magic 20.0 factor into a constant?

Also extremly nit (so nit that I will except not fixing it), but it is preferable to multiply with 0.05 (as division is technically more expensive).